tst/PortingAssistantExtensionUITests/ControlsTest.cs

```diff
@@ -27,6 +27,8 @@ namespace PortingAssistantExtensionUITests
     [TestClass]
     public class ControlsTest : VisualStudioSession 
     {
+        public ControlsTest() : base(@"C:\Program Files (x86)\Microsoft Visual Studio\2019\Professional\Common7\IDE\devenv.exe") { }
```

Comment:
try to check if the file exist before run.

Reply:
Another thing that might be good to consider is community and enterprise? Not sure if we can encapsulate all that logic together?

Reply:
The check if the file exists has been added to the TestInitialization. 
Regarding the type of installation, it should be uniform across test machines/codebuild or we'll need to have a logic to find the correct VS installation as well as preference if multiple versions exist on the machine before running the tests.